src/eko/evolution_operator/grid.py

```diff
@@ -254,14 +259,9 @@ def generate(self, q2):
             )
             a_s = sc.a_s(op.q2_to / self.config["fact_to_ren"], op.q2_to)
 
-            # compute exact inverse if necessary
-            if self.config["backward_inversion"] == "exact" and is_backward:
-                if self.config["intrinsic_range"] is None:
-                    # compute inverse exact
-                    ome.compute(a_s)
-                else:
-                    # compute intrinsic inverse exact
-                    ome.compute(a_s, nf=op.nf)
+            # compute the matching if not done yet
+            if self.ome_members is None:
+                ome.compute(a_s)
```

Comment:
Since the matching are done at threshold then it would be a good idea to group the computation together with the threshold operator, packing inside the function `get_threshold_operators()`, and in this way calling `ome.compute()` in a single place

Reply:
This is also a good suggestion! Thanks